File: api/entities/Patient.cs
namespace api.entities
{
    public class Patient
    {
        public int Id { get; set; }
        public float height { get; set; }
        public float weight { get; set; }
        public int age { get; set; }
        public string gender { get; set; }
        public string FEV1 { get; set; }
        public string TLC { get; set; }
        public string RV { get; set; }
        public string ERV { get; set; }
        public string IC { get; set; }
        public string VC { get; set; }
        public virtual GLI GLI {get; set;}
        public virtual CAS CAS {get; set;}

    }
}

File: client/src/app/_models/patient.ts
import { CAS } from "./cas";
import { GLI } from "./gli";

export interface Patient {
    id: number;
    height: number;
    weight: number;
    age: number;
    gender: string;
    feV1: string;
    tlc: string;
    rv: string;
    erv: string;
    ic: string;
    vc: string;
    gli: GLI;
    cas: CAS;
}